src/utils/calculateTooltipOffset.test.js

```diff
@@ -2,14 +2,14 @@ import calculateTooltipOffset from './calculateTooltipOffset';
 
 describe('Tooltip offset calculator', () => {
   it('should calculate offset for a 2 hour meeting', () => {
-    expect(calculateTooltipOffset(2)).toBe(-54);
+    expect(calculateTooltipOffset(2)).toBe(-54.5);
```

Comment:
i noticed it. The problem is that this test does not contain any comments about "why do we expect -54.5"